sdk/digitaltwins/Azure.DigitalTwins.Core/tests/Record-Tests.ps1

```diff
@@ -0,0 +1,53 @@
+﻿param(
+    [Parameter()]
+    [string] $TestName
+)
+
+$user = $env:UserName
+$fileName = "$user.config.json"
+$oldConfig = Get-Content -Path $PSScriptRoot/config/$fileName
+
+$newConfig = @"
+{
+    "TestMode":  "Record"	
+}
+"@
+
+Write-Host("Updating user config file to Record test mode`n")
+$newConfig | Out-File "$PSScriptRoot\config\$fileName"
+
+if (-not $TestName)
+{
+    # Run all tests
+    dotnet test
+}
+else
+{
+    # Run only specified test
+    dotnet test --filter FullyQualifiedName~$TestName
```

Comment:
Doesn't this need to have the namespace info as well?

Reply:
I see that the `~` operator can accept just the filename: `dotnet test --filter "FullyQualifiedName~UnitTest1"`

Reply:
Nope because ~ means contains. However, we could do = with namespace. But if we change the namespace, we will have to remember to update this as well. I don't have a very strong opinion either way.